styles.css

```diff
@@ -0,0 +1,104 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@600&family=Roboto:wght@400&display=swap");
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+:root {
+  --ff-primary: "Roboto", sans-serif;
+  --ff-secondary: "Montserrat", sans-serif;
+  --clr-primary: #fe4775;
+  --clr-secondary: #161060;
+  --clr-text-primary: #393939;
+  --clr-text-secondary: #111;
+  --clr-text-passive: hsla(0, 0%, 0%, 0.55);
+  --clr-content-passive: #fbfbfb;
+  --fw-primary: 400;
+  --fw-secondary: 600;
+}
+
+body {
```

Comment:
I think we should set `color` here to `--clr-text-primary`, I forgot to do that in the `global.css` for the components. There is an issue for that #17